src/sql/workbench/contrib/welcome/page/browser/gettingStartedTour.css

```diff
@@ -0,0 +1,371 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.ads_tour:focus {
```

Comment:
You've added a bunch of icons but most of them don't seem to be used - is this intentional?

Reply:
I did find a few images that weren't being used anymore. Most of them are being used in the gettingStartedTour.ts. If I have missed any could you point to one example so I can see what is I am missing.